### PR TITLE
Add samples for fhir and hl7 libs

### PR DESCRIPTION
**Pull request by nirmal070125**
These samples will be pointed out from the webpage in ballerina.io.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: working_with_fhir/working_with_fhir_parsing.bal [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import ballerinax/health.fhir.r4.parser as parser;
```
Comment (ThishaniLucas) [inline working_with_fhir/working_with_fhir_parsing.bal:1]: Any advantages of having this alias? The prefix is anyway `parser`.

File: working_with_fhir/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[package]
+org = "healthcare_samples"
+name = "working_with_fhir"
+version = "0.1.0"
+distribution = "2201.4.0"
```
Comment (Thiyanwso2) [inline working_with_fhir/Ballerina.toml:5]: why 2201.4.0? Currently we are using 2201.4.1

File: working_with_fhir/working_with_fhir_parsing.bal (hunk 1 above)
Comment (nirmal070125) [reply working_with_fhir/working_with_fhir_parsing.bal:1]: Typo.

File: working_with_fhir/Ballerina.toml (hunk 2 above)
Comment (nirmal070125) [reply working_with_fhir/Ballerina.toml:5]: Ack.

**Review by ThishaniLucas — APPROVED** [review]

File: working_with_hl7/working_with_hl7_parsing.bal [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import wso2healthcare/healthcare.hl7 as hl7;
```
Comment (sameeragunarathne) [inline working_with_hl7/working_with_hl7_parsing.bal:2]: shall we change to ballerinax dependencies and check since the new changes are released?

Comment (nirmal070125) [reply working_with_hl7/working_with_hl7_parsing.bal]: updated.

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**